index.bs

```diff
@@ -484,169 +633,30 @@ dictionary QuicTransportStats {
 
 The dictionary SHALL have the following attributes:
 
-: <dfn for="QuicTransportStats" dict-member>timestamp</dfn>
+: <dfn for="WebTransportStats" dict-member>timestamp</dfn>
 :: The `timestamp` for when the stats are gathered, relative to the
```

Comment:
Should use DOMTimeStamp: https://heycam.github.io/webidl/#DOMTimeStamp

Reply:
Is there any specific reason for it to use low-res version? https://w3c.github.io/webrtc-pc/#dom-rtcstats uses `DOMHighResTimeStamp` too.

Reply:
Sorry, yes, `DOMHighResTimeStamp` is the right choice.